recipe.ts

```diff
@@ -0,0 +1,53 @@
+import { Ingredient } from "./ingredient";
+import { ArrayLengthGreaterThan, Int, StringMinLength } from "./validator";
+
+export class Recipe {
+  @Int()
+  private _id!: number;
+  @StringMinLength(1)
```

Comment:
really grasped the use of decorators by seeing this example, so thank you – as always very enlightening to see your code :)

Reply:
Thanks! 😇 Biggest breakthrough for me was just making sure I was using the correct type of decorator for what I needed 🤦‍♀️